src/Directory.Build.props

```diff
@@ -1,5 +1,404 @@
 <Project>
-  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory).., Directory.Build.props))\Directory.Build.props" />
+
+  <!-- TODO: Remove condition and Arcade SDK import below when consolidated. -->
+  <PropertyGroup>
+    <IsRuntimeRepository Condition="Exists('..\..\.dotnet-runtime-placeholder')">true</IsRuntimeRepository>
+  </PropertyGroup>
+  <Import Project="..\..\Directory.Build.props" Condition="'$(IsRuntimeRepository)' == 'true'" />
+
+  <!--
+    Import the arcade sdk with these requirements.
+    After:
+      Configuration setup
+    Before:
+      DebugType needs to be not be set to embedded at least for facades
+      OutputPath setup
+  -->
+  <Import Project="Sdk.props" Sdk="Microsoft.DotNet.Arcade.Sdk" Condition="'$(IsRuntimeRepository)' != 'true'" />
+
+  <!--
+    Get ProjectToBuild and '<subset>ProjectToBuild' items. Using the items lets projects handle
+    $(Subset) automatically when creating project-to-project dependencies.
+  -->
+  <Import Project="$(RepositoryEngineeringDir)Build.props" />
+
+  <!--
+    Before Microsoft.Common.targets, set the extensions path to match the restore dir as Arcade
+    sets it, so MSBuild packages with targets files will be found and imported.
+  -->
+  <PropertyGroup>
+    <MSBuildProjectExtensionsPath>$(BaseIntermediateOutputPath)</MSBuildProjectExtensionsPath>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <ArcadeSdkMSBuildProjectDir>$([System.IO.Path]::GetDirectoryName('$(ArcadeSdkBuildTasksAssembly)'))\..\</ArcadeSdkMSBuildProjectDir>
+    <ArcadeSdkSignProject>$(ArcadeSdkMSBuildProjectDir)Sign.proj</ArcadeSdkSignProject>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(MSBuildProjectExtension)' == '.csproj'">
+    <!-- SDK-based projects' Platform should default to AnyCPU, not $(TargetArchitecture) like Directory.Build.props sets. -->
+    <Platform>AnyCPU</Platform>
+    <UsingNETSdkCompiler>true</UsingNETSdkCompiler>
+    <DisableBuildToolsRoslynVersion>true</DisableBuildToolsRoslynVersion>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <InstallerProjectRoot>$(MSBuildThisFileDirectory)</InstallerProjectRoot>
+    <SigningToolsDir>$(InstallerProjectRoot)signing\</SigningToolsDir>
+    <LicenseFile>$(RepoRoot)LICENSE.TXT</LicenseFile>
+  </PropertyGroup>
+
+  <!-- TODO: Remove when consolidated. -->
```

Comment:
I'd recommend a specific marker that can easily be found with a global search and burned down:

```
<!-- TODO: (Consolidation) Remove when consolidated. -->
```

(`TODO: (Consolidation)` being the search term there. In this case the rest of the comment is a little redundant, but there's room for some freeform extra info where it makes sense.)